airflow/providers/amazon/aws/transfers/sql_to_s3.py

```diff
@@ -148,9 +148,13 @@ def _fix_dtypes(df: DataFrame, file_format: FILE_FORMAT) -> None:
         for col in df:
 
             if df[col].dtype.name == "object" and file_format == "parquet":
-                # if the type wasn't identified or converted, change it to a string so if can still be
-                # processed.
-                df[col] = df[col].astype(str)
+                # if the type wasn't identified or converted, try parsing it to datetime
+                try:
+                    df[col] = pd.to_datetime(df[col])
+                    df[col].fillna(pd.to_datetime('1970-01-01'),inplace=True)
```

Comment:
Can you give more details on this one? Why doing that? I am pretty sure there is a valid reason but please add it in comments to help people understand it :)

Reply:
Because - '1st January 1970' usually called as "epoch date" is the date when the time started for Unix computers.

Reply:
Ok but could you add a comment like "Fill this cell with 01/01/70 in case of None value". That's my understanding of what this line does but I am not sure.